src/components/Navbar.js

```diff
@@ -39,6 +37,33 @@ function Navbar() {
 							"md:flex md:flex-row md:justify-center md:w-auto md:relative xs:hidden xs:flex-none"
 						}
 					>
+						<div className="p-2 mx-2 dropdown inline-block relative">
+							<button>
+								Courses
+								<FontAwesomeIcon
+									className="ml-2"
+									icon={faAngleDown}
+								/>
+							</button>
+							<ul className="dropdown-menu absolute hidden pt-2 bg-white shadow-2xl">
+								<Link to="/courses/python/beginner">
+									<li className="rounded-t py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
+										Python Beginner
+									</li>
+								</Link>
+								<Link to="/courses/python/intermediate">
+									<li className="py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
+										Python Intermediate
+									</li>
+								</Link>
+								<Link to="/courses/python/advanced">
+									<li className="rounded-b py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
+										Python Advanced
+									</li>
+								</Link>
```

Comment:
you should probably have this data in `navbarItems` instead of hard coding the sub-menu like this.

e.g.:
```js
export const navbarItems = [
  {
    display: "About Us",
    to: "/aboutus",
  },
  {
    display: "Courses",
    to: "/courses",
    subitems: [
      {
        display: "Python Beginner",
        to: "/courses/python/beginner",
      },
      .
      .
      .
    ],
  },
];
```